src/index.js

```diff
@@ -38,6 +39,10 @@ function hydrateApp(kcInfo) {
 
   const store = createStore({ ...window.__data, auth: kcInfo }, history);
 
+  // optional, but required for refetchOnFocus/refetchOnReconnect behaviors
+  // see `setupListeners` docs - takes an optional callback as the 2nd arg for customization
+  setupListeners(store.dispatch);
```

Comment:
Does this influence other issues not only simple-mode?

Reply:
No. It doesn't. It doesn't even influence simple-mode as it doesn't use refetchOnFocus/refetchOnReconnect behaviors. It is here for "not to forget when will start using it", but setup will be far in the past, so nobody will remember.